src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -277,16 +276,12 @@ public async Task<(string tabLocationHeader, IconSource tabIcon, string toolTipT
 			return (tabLocationHeader, iconSource, toolTipText);
 		}
 
-		public async Task OnNavigatedTo(NavigationEventArgs e)
+		public async Task OnNavigatedTo(object? parameter)
 		{
-			if (e.NavigationMode == NavigationMode.Back)
-				return;
-
```

Comment:
Why was this removed?

Reply:
I don't see any cases where the (whole) app view would be navigated backward. We only navigated to the splash screen and then the main view